ext/ox/gen_load.c

```diff
@@ -343,8 +455,24 @@ end_element(PInfo pi, const char *ename) {
 static void
 add_instruct(PInfo pi, const char *name, Attr attrs, const char *content) {
     VALUE       inst;
-    VALUE       s = rb_str_new2(name);
-    VALUE       c = Qnil;
+    VALUE       s;
+    VALUE       c;
```

Comment:
Please initialize with `Qnil`. If not interned and content is NULL then c will not be set.